pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -396,6 +398,23 @@ func (pr *PipelineRunStatus) MarkRunning(reason, messageFormat string, messageA
 	pipelineRunCondSet.Manage(pr).MarkUnknown(apis.ConditionSucceeded, reason, messageFormat, messageA...)
 }
 
+// ChildStatusReference is used to point to the statuses of individual TaskRuns and Runs within this PipelineRun.
+type ChildStatusReference struct {
+	runtime.TypeMeta `json:",inline"`
+	// Name is the name of the TaskRun or Run this is referencing.
+	Name string `json:"name,omitempty"`
+	// PipelineTaskName is the name of the PipelineTask this is referencing.
+	PipelineTaskName string `json:"pipelineTaskName,omitempty"`
+
+	// ConditionChecks is the the list of condition checks, including their names and statuses, for the PipelineTask.
+	// Deprecated: This field will be removed when conditions are removed for v1.
```

Comment:
nit: conditions may be removed in v1beta1 as well according to our compatibility policy so I'd just remove the "v1" bit

Reply:
👍